packages/harpguru-core/src/global.d.ts

```diff
@@ -33,17 +33,19 @@ declare module 'reactn/default' {
     readonly columnBounds: ColumnBounds
     readonly tuningId: TuningIds
     readonly valvingId: ValvingIds
-    readonly activeInteractionMatrix: HarpFaceMatrix<Interaction>
-    readonly activeDegreeMatrix: HarpFaceMatrix<Degree>
-    readonly activePitchMatrix: HarpFaceMatrix<Pitch>
+    readonly activeInteractionMatrix: HarpFaceMatrices<Interaction>
+    readonly activeDegreeMatrix: HarpFaceMatrices<Degree>
+    readonly activePitchMatrix: HarpFaceMatrices<Pitch>
     readonly activeDegreeIds: ReadonlyArray<DegreeIds>
     readonly activePitchIds: ReadonlyArray<PitchIds>
     readonly pozitionId: PozitionIds
     readonly rootPitchId: PitchIds
     readonly harpKeyId: PitchIds
-    readonly viewableInteractionMatrix: HarpFaceMatrix<Interaction>
-    readonly viewableDegreeMatrix: HarpFaceMatrix<Degree>
-    readonly viewablePitchMatrix: HarpFaceMatrix<Pitch>
-    readonly layoutFacts: LayoutFacts
+    readonly viewableInteractionMatrix: HarpFaceMatrices<Interaction>
+    readonly viewableDegreeMatrix: HarpFaceMatrices<Degree>
+    readonly viewablePitchMatrix: HarpFaceMatrices<Pitch>
+    // TODO: It would be nice to have a type which is more in keeping with the
```

Comment:
TODO here. Yes this would be a good idea. `HarpFacePossiblePair` perhaps.

Reply:
Covered off in https://github.com/js-jslog/harpguru/pull/130. As acknowledged in https://github.com/js-jslog/harpguru/issues/131#issuecomment-981960861, the name `HarpFaceFacts` could be a bit better, but the improvements still stand with that name so I'll close this discussion.